Makefile

```diff
@@ -6,6 +6,8 @@ DEV_CONTAINER_VENV_VOLUME:=concrete-ml-venv
 DEV_CONTAINER_CACHE_VOLUME:=concrete-ml-cache
 DOCKER_VENV_PATH:="$${HOME}"/dev_venv/
 SRC_DIR:=src
+TEST:=tests
+N_CPU:=4
```

Comment:
What is this used for? pytest? should we really hardcode this?

Reply:
Alright then. It's just that this 4 looks really arbitrary but if we were doing this before then fine.

Reply:
yes it is, but there's a coment in the pytest command that sayd that we need to set a low number (lower than the machine's number of CPUs), otherwise it fails : 
> because most tests include parallel execution using all CPUs and too many parallel tests would lead to contention. Thus N is set to something lower than the overall number of CPUs